### DISCUSSION TITLE
how can i get the asset's value

### DISCUSSION THREAD

**Discussion by headsome911** [dagster-io/dagster#5225] [Q&A] [answered]
suppose that: i have two pipeline (named A /B);
in A: I create a asset named 'datasets' 
![image](https://user-images.githubusercontent.com/26199251/137618356-f76f85f1-9bb6-4338-b5b0-cf40a7b73a58.png)
the value is defined as:
![image](https://user-images.githubusercontent.com/26199251/137618378-cfee2fed-a6af-467f-a2aa-02157dbf0c67.png)

in B : I create a asset_sensor(monitor 'datasets' asset created by A). how can i get the value of 'datasets'

**Comment by little-hang** [comment, marked as answer]
hi, this is my method。
suppose that: 
(1)I create a asset_sensor as follow:
![image](https://user-images.githubusercontent.com/24517540/137692263-2a1cb9ac-854a-45c9-8a6a-04d0f4ef444d.png)
(2) then I get the metadata of the AssetMaterialization:
![image](https://user-images.githubusercontent.com/24517540/137692496-3965e2ea-a788-4147-98ea-987c65e2a1f2.png)
 and we can get the 'datasets' = {'1_100': 9481, '2_200': 111}

**Reply by headsome911** [reply]
Thank you!

**Comment by little-hang** [comment]
you can also create sensor as follow:
![image](https://user-images.githubusercontent.com/24517540/137700006-0b8215ce-7451-442a-87c6-3f83fd1179d3.png)
 and then ,use the same method to get the asset's metadata as above。
